coffea/nanoaod/methods/jets.py

```diff
@@ -4,6 +4,7 @@
 
 class FatJet(LorentzVector):
     '''NanoAOD large radius jet object'''
+    _enable_genjet = False
```

Comment:
I guess we should add at some point a version specifier to the NanoEvents constructor to turn on these features for known good NanoAOD versions. The only reason the enable flag exists rather than simply checking for existence of the index map is because in v5 the index map had a bug and some indices were invalid (> length of genjet).

Reply:
Yeah the flag is not great and afaict not documented so it was a bit of digging at first.